Email DT fix (#33924)
demisto/content: Packs/MISP/ReleaseNotes/2_1_43.md

#### Integrations

##### MISP v3

> Fixed an issue in `Base` pack (Version `1_33_52`) so now MISP will correctly input email addresses into context under `Accounts.Email` and not under `Email`.

Reviewer: ```suggestion
Fixed an issue in `Base` pack (Version `1.33.52`) so now MISP will correctly input email addresses into context under `Accounts.Email` and not under `Email`.
```

Reviewer: Can we mention the specific command/s that is impacted?

> Due to a fixed issue in `Base` pack (Version `1_33_52`) MISP will now correctly input email addresses into context under `Accounts.Email` and not under `Email` as it did up until now.

Reviewer 2: ```suggestion
Fixed an issue in `Base` pack (Version `1_33_52`) so now MISP will correctly input email addresses into context under `Accounts.Email` and not under `Email`.
```
